packages/calcite-components/src/components/functional/ExpandToggle.tsx

```diff
@@ -62,7 +62,7 @@ const setTooltipReference = ({
   return referenceElement;
 };
 
-export const ExpandToggle: FunctionalComponent<ExpandToggleProps> = ({
+export const ExpandToggle: FunctionalComponent<ExpandToggleProps & any> = ({
```

Comment:
`ExpandToggleProps` should extend `JSXBase.HTMLAttributes` instead of using `any` here (see [example](https://github.com/Esri/calcite-components/blob/14036c1354419222bc5b839af8b68675b17ebf62/packages/calcite-components/src/components/functional/Heading.tsx#L6-L8)).